pkg/services/grafana-apiserver/service.go

```diff
@@ -251,9 +251,6 @@ func (s *service) start(ctx context.Context) error {
 		if err := o.Etcd.Validate(); len(err) > 0 {
 			return err[0]
 		}
-		if err := o.Etcd.Complete(serverConfig.Config.StorageObjectCountTracker, serverConfig.Config.DrainedNotify(), serverConfig.Config.AddPostStartHook); err != nil {
```

Comment:
Complete no longer exists -- is it necessary?

Reply:
i ran into this on another branch. it's safe to remove